crypto/s2n_ecc_evp.h

```diff
@@ -35,21 +35,18 @@ struct s2n_ecc_named_curve {
 extern const struct s2n_ecc_named_curve s2n_ecc_curve_secp256r1;
 extern const struct s2n_ecc_named_curve s2n_ecc_curve_secp384r1;
 
-#define S2N_ECC_EVP_SUPPORTED_CURVES_COUNT 2
-
 /* BoringSSL only supports using EVP_PKEY_X25519 with "modern" EC EVP APIs. BoringSSL has a note to possibly add this in
  * the future. See https://github.com/google/boringssl/blob/master/crypto/evp/p_x25519_asn1.c#L233
  */
 #if S2N_OPENSSL_VERSION_AT_LEAST(1, 1, 0) && !defined(LIBRESSL_VERSION_NUMBER) && !defined(OPENSSL_IS_BORINGSSL)
 #define MODERN_EC_SUPPORTED 1
```

Comment:
nit: Can you add some indentation here to make it clear which blocks are defined in which #ifdef?